### PR TITLE
Feature auto scroll 

### PR DESCRIPTION
**Pull request by gurjit03**
* [x]  Passed [sanity tests](http://bit.ly/sttm-sanity-tests).
* [x]  Ran `npm test` & fixed newly introduced lint errors.
* [x]  Checked console for errors.

 [ a - is shortcut ]
In paused state
image

In scrolling state
image

image

<!-- New to markdown? Simply put an 'x' between [ ] to check it. Like [x] this. (No spaces).

### CODE REVIEW

**Comment by saintsoldierx** [comment]
image

**Comment by saintsoldierx** [comment]
image
For mobile devices

**Comment by saintsoldierx** [comment]
@gurjit03 please see designs above for auto scroll.

File: src/scss/_display-controls.scss [hunk 1]
```diff
@@ -12,9 +12,9 @@
   transition: transform 0.2s;
   z-index: 1;
 
-  .dark-mode & {
-    background-color: $sttm-light-black;
-  }
+  // .dark-mode & {
```
Comment (Gauravjeetsingh) [inline src/scss/_display-controls.scss:15]: We need this here, currently, dark mode is like this

image

Comment (gurjit03) [reply src/scss/_display-controls.scss:15]: ahh good catch thanks !

**Comment by saintsoldierx** [comment]
It doesn't seem to work when i press play to scroll. Moves page a little then stops.

Also, I think it may look nice if we change opacity of the entire panel to 0.5 (.floatingActions) and on hover, change it to 1.0 ?

**Comment by gurjit03** [comment]
@saintsoldierx yeah was thinking the same (for the opacity of the floating icons), need to check the scrolling behaviour was working for me and @Gauravjeetsingh.

**Comment by Gauravjeetsingh** [comment]
@gurjit03 I think @saintsoldierx veerji meant, that auto-scroll stops if we manually scroll while the auto-scroll is active.

This, I can reproduce that from my Linux machine as well.

**Pull request merged**